lib/mpl_toolkits/axisartist/grid_finder.py

```diff
@@ -246,9 +232,11 @@ def get_transform(self):
 
     update_transform = set_transform  # backcompat alias.
 
+    @_api.deprecated("3.9", alternative="grid_finder.get_transform()")
```

Comment:
I guess the deprecation will start in 3.11 now.
```suggestion
    @_api.deprecated("3.11", alternative="grid_finder.get_transform()")
```